src/app/core/layout/footer/footer.component.html

```diff
@@ -1 +1,58 @@
-<p>footer works!</p>
+
+<footer class="bg-[#29408d] text-white py-6 px-6  bottom-0 left-0 w-full">
+    <div class="container mx-auto grid grid-cols-1 sm:grid-cols-2 md:grid-cols-4 gap-6">
+        <!-- About Us -->
+        <div>
+            <h2 class="text-2xl font-semibold">About Us</h2>
+            <p class="mt-2 text-sm leading-relaxed">
+                We are a company that cares about your health and helps hospitals 
+                and doctors provide the best equipment and treatment.
+            </p>
+            <p class="mt-4 text-sm font-medium">Follow Us</p>
+            <div class="flex space-x-3 mt-3">
+                <a href="#" class="text-xl hover:text-gray-300 transition"><i class="fab fa-facebook"></i></a>
```

Comment:
```suggestion
                <a href="#" class="text-xl hover:text-gray-300 transition"><i class="fab fa-facebook"></i></a>
```
store this  text-gray-300  in globle style and used